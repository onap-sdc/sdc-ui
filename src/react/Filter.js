import React from 'react';
import PropTypes from 'prop-types';

const Filter = ({ className, children }) => (
    <div className={`sdc-filter ${className}`}>
        {children}
    </div>
);

Filter.propTypes = {
    className: PropTypes.string,
    children: PropTypes.node
};

Filter.defaultProps = {
    className: ''
};
export default Filter;