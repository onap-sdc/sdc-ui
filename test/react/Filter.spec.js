import React from 'react';
import Filter from '../../src/react/Filter.js';

import renderer from 'react-test-renderer';

describe('Filter', () => {
    test('Filter - Default', () => {
        const filter = renderer.create(<Filter>Filter</Filter>).toJSON();
        expect(filter).toMatchSnapshot();
    });
});
