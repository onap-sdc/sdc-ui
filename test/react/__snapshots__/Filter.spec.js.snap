// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter Filter - Default 1`] = `
<div
  className="sdc-filter "
>
  Filter
</div>
`;
